src/main/html/index.html

```diff
@@ -128,12 +128,11 @@
           // http://highlightjs.readthedocs.io/en/latest/api.html#inithighlighting
           hljs.initHighlighting()
 
-          if (typeof initOAuth == "function" && !!window.anthony) {
+          if (typeof initOAuth == "function" && (!accessToken || !!window.clientId)) {
             // NG-4689: Parameters are required by the initOAuth function to
             // properly open the OAuth dialog.  They are derived from
             // https://github.com/swagger-api/
-            // Hack: really hacky way to set the audience while we integrate the catalog api
-            var audience = getQueryParam("audience") || (window.location.host.match(/staging|integration|localhost/) ? "https://development.api.cimpress.io/" : "https://api.cimpress.io/");
+            var audience = window.audience || (window.location.host.match(/staging|integration|localhost/) ? "https://development.api.cimpress.io/" : "https://api.cimpress.io/");
```

Comment:
Since this is open source, it probably doesn't make sense to include our internal resource servers as defaults. Shouldn't we just require the caller to be explicit about the audience?